bundles/org.openhab.binding.lutron/README.md

```diff
@@ -416,23 +420,44 @@ The only exceptions are **greenmode** *step*, which is periodically polled and a
 Many other channels accept REFRESH commands to initiate a poll, but sending one should not normally be necessary.
 
 
-## Full Radio RA2 Configuration Example
+## Radio RA2 Configuration File Example
 
-demo.Things:
+demo.things:
 
 ```
-lutron:dimmer:theater (lutron:ipbridge:radiora2) [ integrationId=8, fadeOutTime=2 ]
-lutron:occupancysensor:theater (lutron:ipbridge:radiora2) [ integrationId=9 ]
-lutron:keypad:theater (lutron:ipbridge:radiora2) [ integrationId=10 ]
+Bridge lutron:ipbridge:radiora2 [ ipAddress="192.168.1.200", user="lutron", password="integration" ] {
+        Thing dimmer lrtable "Table Lamp" @ "Living Room" [ integrationId=45, fadeInTime=0.5, fadeOutTime=5 ]
+        Thing dimmer lrtorch "Torch Lamp" @ "Living Room" [ integrationId=44, fadeInTime=0.5, fadeOutTime=5 ]
+        Thing dimmer lrspot [ integrationId=38, fadeInTime=0.5, fadeOutTime=5 ]
+        Thing switch path [ integrationId=61 ]
+        Thing keypad entrykeypad [ integrationId=64, model="W7B", autorelease=true ]
+        Thing ttkeypad bedroomkeypad [ integrationId=28, model="T15RL", autorelease=true ]
+        Thing pico librarypico [ integrationId=71, model="3BRL", autorelease=true ]
+        Thing vcrx vcrx1 [ integrationId=34, autorelease=true ]
+        Thing cco garage1 [ integrationId=75, outputType="Pulsed", pulseLength=0.5 ]
+        Thing shade libraryshade1 [ integrationId=66]
+        Thing greenmode greenmode [ integrationId=22 ]
+        Thing timeclock timeclock [ integrationId=23 ]
+        Thing occupancysensor laundryocc [ ]
```

Comment:
Do we need a configuration parameter "integrationId" here?

Reply:
Yes.  That's what I get for changing something at the last minute without actually concentrating on what I'm doing. Thanks!